packages/vega/test/scenegraphs/arc.json

```diff
@@ -21,7 +21,9 @@
               "startAngle": 0,
               "endAngle": 0.474202664692799,
               "innerRadius": 20,
-              "outerRadius": 58.43075691322092
+              "outerRadius": 58.43075691322092,
+              "ariaRole": "graphics-symbol",
```

Comment:
While the latest revisions have helpfully limited the aria annotations produced for guides, we can still generate a lot of annotations, one for each standard mark. Bloat issues aside, is this really what we want to do? Is this what is most helpful?

An alternative might be to emit standard aria annotations not for each mark scenegraph item, but once for the mark group itself. In other words: put the annotation on the mark parent in the scenegraph (not the item entries) and then have the renderers write this annotation to the enclosing SVG `g` element, rather than the individual elements (e.g., `path`, `text`, etc.) corresponding to mark items.

Would that make sense to do?

Reply:
I think that makes sense for line, trail, and area marks. For rect marks or text marks, there is value for a screen reader user to be able to navigate the individual items. How do you think we can get the best of both worlds?